<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>make_order</class>
 <widget class="QWidget" name="make_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оформление заказа</string>
  </property>
  <widget class="QPushButton" name="btn_cart">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Корзина</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_to_cart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Добавить в корзину</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search_order">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Поиск...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_quantity">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_quantity">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>321</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Колличество:</string>
   </property>
  </widget>
  <widget class="QTableView" name="table_make_order">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>680</width>
     <height>290</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
